tools/topology/topology2/cavs-mixin-mixout-hda.conf

```diff
@@ -69,6 +73,7 @@ Object.Pipeline {
 		direction	capture
 
 		Object.Widget.copier."1" {
+			dai_index 0
```

Comment:
do we ever have a case where dai_type and copier_type are different

Reply:
copier_type is applicable to all copiers (host, module, DAI copiers) whereas dai_type is only applicable for the DAI copiers. When set, the dai_type and copier_type must match. But we need 2 attributes because the dai_type is added as a token whereas copier_type is purely for the use of naming the copier